charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -133,6 +133,10 @@ data:
                 {{ if $account.readPermissions }}--read-permissions {{ join " --read-permissions " $account.readPermissions }}{{ end }} \
                 {{ if $account.writePermissions }}--write-permissions {{ join " --write-permissions " $account.writePermissions }}{{ end }} \
                 --provider-version v2
+    {{- range $index, $customResource := .Values.kubeConfig.customResources }}
+    $HAL_COMMAND config provider kubernetes account \$PROVIDER_COMMAND {{ $name }} --docker-registries dockerhub \
+                --context {{ $account.context }} --provider-version v2 --add-custom-resource={{ $customResource.kubernetesKind }} --spinnaker-kind={{ $customResource.spinnakerKind }} --no-validate
```

Comment:
Can we forego the options `--docker-registries`, `--context` and `--provider-version` as they were already configured above?